AdobeStockImage/etc/di.xml

```diff
@@ -8,4 +8,8 @@
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
     <preference for="Magento\AdobeStockImageApi\Api\Data\ImageInterface" type="Magento\AdobeStockImage\Model\Image"/>
     <preference for="Magento\AdobeStockImageApi\Api\GetImageListInterface" type="Magento\AdobeStockImage\Model\GetImageList"/>
+    <type name="Magento\Catalog\Model\Product\Gallery\Processor">
+        <plugin name="adobe_stock_image_remove_metadata" type="Magento\AdobeStockImage\Plugin\Product\Gallery\Processor" sortOrder="1" disabled="false" />
```

Comment:
Just to ensure there is some flexibility to add plugins before
```suggestion
        <plugin name="adobe_stock_image_remove_metadata" type="Magento\AdobeStockImage\Plugin\Product\Gallery\Processor" sortOrder="10" disabled="false" />
```

Reply:
I added this change